python/nano/src/bigdl/nano/pytorch/torch_nano.py

```diff
@@ -35,13 +36,28 @@
 
 
 class _TorchNanoModule(_LiteModule):
+    def __init__(self, module, precision_plugin, channels_last) -> None:
+        super().__init__(module, precision_plugin)
+        self.channels_last = channels_last
+
     def __getattr__(self, name: str):
         # automatically unwrap attributes access of _LiteModule
         try:
             return super().__getattr__(name)
         except AttributeError:
             return getattr(self.module, name)
 
+    def forward(self, *args: Any, **kwargs: Any) -> Any:
+        """Casts all inputs to the right memory format."""
+        if self.channels_last:
+            def _convert_to_channels_last(t: torch.Tensor) -> torch.Tensor:
+                if t.dim() == 4:
+                    return t.to(memory_format=torch.channels_last)  # type: ignore
+                return t
+            args, kwargs = apply_to_collection([args, kwargs], function=_convert_to_channels_last,
+                                               dtype=torch.Tensor)
+        return super().forward(*args, **kwargs)
```

Comment:
I could not remember if this(i.e. `forward(*args, **kwargs)`) may cause a problem for quantization/trace, could please have a quick check?

The structure for test code could be
```python
# define a nanotorch model
t_model = Trainer.trace(model, ...)
t_model(x)
q_model = Trainer.quantize(model, ...)
q_model(x)
```

Reply:
> I could not remember if this(i.e. `forward(*args, **kwargs)`) may cause a problem for quantization/trace, could please have a quick check?

yes, it reported an error in my test, but will we use `Trainer` and `_TorchNanoModule` together ?

Reply:
Yeah I think. if a user have a pytorch nnmodule and he/she change it to a torchnano model for training acceleration, then, he/she wants to accelerate the model for inference.

Reply:
Should we add the example in this PR or a follow up PR?